src/index.html

```diff
@@ -10,23 +10,40 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search bar airbnb</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="./style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="big-searchbar"
+      action="http://localhost:8080/"
+      method="post"
+      data-qa="big"
+    >
+      <input
+        class="input input--big"
+        type="search"
+        placeholder="Try “Los Angeles“"
+        pattern="\w+"
+        data-qa="keypress"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
```

Comment:
You should also add a class to target this form with the special "sauce" that you used in the input